mod-orders-storage/schemas/po_line.json

```diff
@@ -131,6 +131,11 @@
       "type": "boolean",
       "default": false
     },
+    "isBindaryActive":{
+      "description": "Indicates that this POL is designated as required binding",
+      "type": "boolean",
+      "default": false
+    },
```

Comment:
I think we need to add it to details.json object that should match with PO Line Details
![image](https://github.com/folio-org/acq-models/assets/25097693/5b374698-63eb-497b-a300-f96551521fe7)
Also base requirement in this story https://folio-org.atlassian.net/browse/UIOR-1211 we should locate this new field in Po Line details.
I think it is added to implify common PO Line schema

Reply:
I am writing the validation so, in case of someone send request to /order/order-lines with unexpected request body. As far as I know it will first come as composite poline and converted to poLine. So I wonder it is okay to leave comPoLine. 

I am testing locally, api issue is saying non recognizable field ComPoLine

Reply:
Thanks, I will try to add details.json